Assets/Scripts/Controllers/TimeManager.cs

```diff
@@ -190,4 +190,12 @@ private void InvokeEvent(Action<float> eventAction, float time)
             eventAction(time);
         }
     }
+
+    /// <summary>
+    /// Destroy this instance.
+    /// </summary>
+    public void Destroy()
```

Comment:
Reset might imply to go to a previous state/instance/starting point. Where this just destroys the static reference.